lib/repo/repository.go

```diff
@@ -33,14 +30,19 @@ const (
 	Stable = "stable"
 )
 
+// baseDir is a directory of iotit related files and configurations
 var baseDir = filepath.Join(help.UserHomeDir(), ".iotit")
-var imageDir = filepath.Join(baseDir, "images")
-var vboxDir = filepath.Join(baseDir, "virtualbox")
+
+// vboxDir is a directory of virtualboxes
```

Comment:
__[golint]__
* [golint] comment on exported var VboxDir should be of the form "VboxDir ..."

<!-- Comment by [SideCI](https://sideci.com) -->